### DISCUSSION TITLE
Codemagic in shows error like Request is unauthorized (401), Please check your credentials to access

### DISCUSSION THREAD

**Discussion by sagarkothiya80** [codemagic-ci-cd/codemagic-docs#2870] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

Cannot access the repository
Request is unauthorized (401). Please check your credentials to access

### Expected results

resolution

### Actual results

resolution

**Comment by dam-ease** [comment, marked as answer]
Hello @sagarkothiya80 
The possible solutions to this issue are highlighted in detail in this section of the docs -https://docs.codemagic.io/troubleshooting/common-issues/#cannot-access-the-repository-request-is-unauthorized-401. Please can you check this out and let me know how it goes on your end?

**Reply by dam-ease** [reply]
Hello @sagarkothiya80.
Have you been able to try the solution out?
Is all good now?
Please if you have any more blockers or issues, kindly let me know. Glad to help!
